driver/xrt/include/accl/coyotedevice.hpp

```diff
@@ -127,7 +127,7 @@ class CoyoteDevice : public CCLO {
   // The CCLO kernel is still managed by coyote_proc
   unsigned int num_qp;
   std::vector<fpga::cProcess*> coyote_qProc_vec;
-
+  std::vector<std::shared_ptr<fpga::cProcess>> coyote_all_cprocs;
```

Comment:
we already have a vector of cprocesses, coyote_qproc_vec. Can't you use the existing vector?

Reply:
I'll apply such a fix